src/main/java/com/luv2code/ecommerce/controller/CheckoutController.java

```diff
@@ -16,14 +16,17 @@
 import com.stripe.exception.StripeException;
 import com.stripe.model.PaymentIntent;
 
+// comments
 @RestController
 @RequestMapping("/api/checkout")
 public class CheckoutController {
 
+    // comments
     private Logger logger = Logger.getLogger(getClass().getName());
 
     private CheckoutService checkoutService;
 
+    // comments
     public CheckoutController(CheckoutService checkoutService) {
```

Comment:
Ovakve konstruktore izbjegavaj to mozes rjesiti na dva nacina.

1. Koristi `@Autowired` anotiaciju na `CheckoutService`
2. (ovo je preferirano) dodaj `@RequiredArgsConstructor` i `CheckoutService` promjeni u private `final`.
`RequiredArgsConstructor` automatski kreira konstruktor za sve nuzno potrebne stvari, npr. `final` klase,